### PR TITLE
Detect tasks changes instead of apps changes

### LINKED ISSUE #5: ZooKeeper event is detected before the changes are applied
When you scale an application, you got the ZK event immediately.
But when you call the API to get the endpoints, it's not up-to-date.

The proxy configuration generated is not correct.

### PR DESCRIPTION
**Pull request by MiLk**
- apps change when you update the number of instances
- tasks change when an instance is modified

If you watch only the app, you will get only the old endpoints.
You have to wait for the new instance.

Fix #5

### CODE REVIEW

**Comment by MiLk** [comment]
@ronkorving Could you review it? Thank you.

**Comment by stelcheck** [comment]
LGTM. @ronkorving good for you?

File: index.js [hunk 1]
```diff
@@ -95,10 +95,24 @@ function listChildren() {
         return;
       }
 
-      children = children.filter(function (child) {
-        return child.search('app') === 0;
+      // List the running apps
+      var apps = children
+        .filter(function (child) {
+          return child.search('app') === 0;
```
Comment (ronkorving) [inline index.js]: you could just use indexOf here, like you do below?

**Comment by ronkorving** [comment]
lgtm.

**Pull request merged**